docs/integrations.md

```diff
@@ -1,9 +1,9 @@
 # Integration
 
-Zafira is integrated with multiple 3rd-party tools like Amazon, Slack, Gmail etc. You are able to add new integration in working Zafira without restart of web application. Navigate to **Username > Integrations** in top navigation menu:
+Zafira is integrated with multiple 3rd-party tools like Amazon, Slack, Gmail etc. You are able to add new integration in working Zafira without restart of web application. Navigate to **Integrations** icon in sidebar menu:
 
 <p align="center">
-  <img src="../img/menu_profile.png">
+  <img src="../img/sidebar_integrations.png">
 </p>
 
 On the integrations page you will find multiple blocks responsible for different integration modules. Pay attention to the fact that security model HTTP/HTTPS may affect your integration URL.
```

Comment:
On the **Integrations** page, you will find several blocks responsible for different integration modules. Please note that the HTTP / HTTPS security model may affect your integration URL.